src/effects/VST3/VST3UIValidator.cpp

```diff
@@ -69,7 +69,6 @@ VST3UIValidator::VST3UIValidator(wxWindow* parent, VST3Wrapper& wrapper, EffectB
 
 VST3UIValidator::~VST3UIValidator()
 {
-   mWrapper.ParamChangedHandler = {};
```

Comment:
mWrapper was a dangling reference?  How did that happen?  I thought the lifetime of the validator was guaranteed to be nested inside that of the instance.

This may be an easy fix, but then the destructor isn't complementing what happened in the constructor.  I want to understand why this line doesn't work.

Reply:
I can't reproduce the problem at e3d0b1ddf9a2363ff3ab65fa23e21e6289fd857d

Reply:
> mWrapper was a dangling reference?

Seem so, I'm experiencing the crash on Windows, 100% reproducible

Reply:
Try this alternative fix:  Move the member `EffectUIHost::mpValidator` after `mpInstance`.

If that fix works, I prefer it.

Reply:
Yes it works, thanks!